packages/trpc-panel/package.json

```
@@ -87,10 +67,34 @@
     "zod": "^3.19.1",
     "zustand": "^4.1.5"
   },
-  "dependencies": {
-    "fuzzysort": "^2.0.4",
-    "path": "^0.12.7",
-    "url": "^0.11.0",
-    "zod-to-json-schema": "^3.20.0"
-  }
+  "directories": {
+    "lib": "lib"
+  },
+  "exports": {
+    ".": {
+      "default": "./lib/index.js",
+      "import": "./lib/index.mjs",
+      "require": "./lib/index.js",
+      "types": "./lib/src/index.d.ts"
+    }
```

This order matters and results in some exports being unable to resolve after build.

```suggestion
  "exports": {
    ".": {
      "types": "./lib/src/index.d.ts",
      "import": "./lib/index.mjs",
      "require": "./lib/index.js",
      "default": "./lib/index.js"
    }
```